source/renderer/app/stores/ProfileStore.js

```diff
@@ -302,10 +304,16 @@ export default class ProfileStore extends Store {
   _acceptTermsOfUse = async () => {
     await this.setTermsOfUseAcceptanceRequest.execute();
     await this.getTermsOfUseAcceptanceRequest.execute();
+    await enableApplicationMenuNavigationChannel.send();
   };
 
-  _getTermsOfUseAcceptance = () => {
-    this.getTermsOfUseAcceptanceRequest.execute();
+  _getTermsOfUseAcceptance = async () => {
+    await this.getTermsOfUseAcceptanceRequest.execute();
+    if (this.getTermsOfUseAcceptanceRequest.result) {
+      await enableApplicationMenuNavigationChannel.send();
+    } else {
```

Comment:
@yakovkaravelov I don't think we need this else clause as the menu navigation is disabled by default?